.yardopts

```diff
@@ -1,5 +1,4 @@
 --title 'AWS X-Ray SDK for Ruby'
 --markup markdown
---markup-provider rdiscount
```

Comment:
This makes sure the API ref doc has the same style as the AWS SDK for Ruby: https://github.com/aws/aws-sdk-ruby/blob/master/.yardopts#L5. Any reason you need to remove it?

Reply:
The rdiscount gem uses the rdiscount C library, which means it's not really JRuby compatible. That's a development dependency, and I couldn't run rake docs (or run default rake task) while it was present. I built docs with the default markup provider and got very similar output (after I fixed the two parsing problems I encountered).